src/main/java/user/requestconnect/IRequestConnectionInput.java

```diff
@@ -1,5 +1,15 @@
 package user.requestconnect;
 
+/**
+ * The inputBoundary of the requestConnect use case, which is implemented in the RequestConnectionInteractor
+ */
 public interface IRequestConnectionInput {
+    /**
+     * The input boundary specifies the RequestConnectionInteractor to have the requestConnection method
+     * when implementing this input boundary
```

Comment:
For documentation you should never have 'this input boundary' or 'it takes' (which I had a lot of in mine, yours looks fine). Usually the first word in the documentation for a method should say what it does, so in this case "Creates a RequestConnectionResponseModel using the input from requestModel", and you shouldn't refer to the method itself by saying this or it.

Reply:
see latest commit